hax/hax/util.py

```diff
@@ -820,6 +835,16 @@ def get_node_ctrl_fids(self,
             list_fids.append(Fid.parse(ctrl_fid))
         return list_fids
 
+    def get_node_hare_motr_s3_fids(self, node: str) -> List[Fid]:
```

Comment:
Docstring is highly desirable given it is a utility function (hence it is intended to be reused in the future)

Reply:
Done.